samples/IISSample/Startup.cs

```diff
@@ -15,6 +15,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
 
             app.Run(async (context) =>
             {
+                app.UseIISPlatformHandler();
```

Comment:
Hmm, how did this get forgotten... Note this should be on line 15 outside the Run.